templates/main.html

```diff
@@ -3,6 +3,14 @@
 
 <script src="{{url_for('static', filename='monsterSort.js')}}"></script>
 <div class='container-fluid monster-display-grid'>
+	<div id='login-success' class="alert alert-success text-center hide">
```

Comment:
Seems mostly duplicated with the below alerts.  Maybe extract them into a partial template?